var/spack/repos/builtin/packages/py-pandas/package.py

```diff
@@ -91,7 +93,7 @@ class PyPandas(PythonPackage):
 
     @property
     def import_modules(self):
-        modules = super().import_modules
+        modules = super(PythonPackage, self).import_modules
```

Comment:
@netromdk I'm a bit surprised that our vermin checks didn't notice this. The former shouldn't be valid in Python 2, right?

Reply:
Yes, you're right. I'm going to add a check for that. Thanks!